index.js

```diff
@@ -275,8 +275,9 @@ function post(msg) {
   http.open("POST", "stdio.html", true);
   http.send(msg);
 }
-// If the address contains localhost, or we are running the page from port 6931, we can assume we're running the test runner and should post stdout logs.
-if (document.URL.indexOf("localhost") != -1 || document.URL.indexOf(":6931/") != -1) {
+
+// If autoRun is enabled, assume we want to automatically close the browser the test(s) finish
+if (autoRun) {
```

Comment:
I don't think the variable `autoRun` exists here? At least looking at this file before, I could not find that as a variable.

How about doing the if check with `if (location.search.includes('autorun')) {`

Reply:
Oh sorry about that, not sure why I thought I had seen an `autoRun` variable when I looked 🤦‍♀️ It seemed to still work in both cases, maybe because of the autoRun function? Anyways I've updated it to add a proper constant now called `autoRunEnabled`